Rakefile

```diff
@@ -46,35 +46,40 @@ def sdks
   }
 end
 
-# See: https://discuss.circleci.com/t/multiple-ios-simulators-available-for-single-os-device-configurations/7854
 def devices
   return {
-    ios: ENV['CIRCLECI'] ? "id='00E25E25-4920-42E1-8CC6-360AEF34C5CE'" : "name='iPhone 6s'",
-    macos: "arch='x86_64'",
-    tvos: ENV['CIRCLECI'] ? "id='31F99277-CC9E-409E-A257-7D1907DECB03'" : "name='Apple TV 1080p'"
+    ios: "OS=#{device_os[:ios]},name=#{device_names[:ios]}",
+    macos: "arch=x86_64",
+    tvos: "OS=#{device_os[:tvos]},name=#{device_names[:tvos]}"
   }
 end
 
-# See: https://discuss.circleci.com/t/xcode-exit-code-65/4284/13
-def uuids
+def device_names
   return {
-    ios: '00E25E25-4920-42E1-8CC6-360AEF34C5CE', # iPhone 6s
-    tvos: '31F99277-CC9E-409E-A257-7D1907DECB03' # Apple TV 1080p
+    ios: "iPhone 6s",
+    tvos: "Apple TV 1080p"
   }
 end
 
-def open_simulator_and_sleep(uuid)
-  return if uuid.nil? # Don't need a sleep on macOS because it runs first.
-  sh "xcrun instruments -w '#{uuid}' || sleep 15"
+def device_os
+  return {
+    ios: "10.2",
```

Comment:
Oh, I didn't know about `latest`. Seems to be a good choice for the `xcodebuild` call.
Is there something similar for `instruments -w`? 

Actually: Do we really need the call to Instruments? 🤔